Apps/W1/HybridGP/app/src/Migration/Support/CheckBooks/GPCheckbookMSTR.table.al

```diff
@@ -208,18 +208,18 @@ table 40099 "GP Checkbook MSTR"
 
     local procedure UpdateBankInfo(BankId: Text[15]; var BankAccount: Record "Bank Account")
     var
-        CMBankMSTR: Record "GP Bank MSTR";
```

Comment:
No problems with this since in the end that change was already made and is in our Feature branch. I agree that it is a better, the correct, naming. It's just that I prefer to limit changes once a code review is started as it makes PR updates more challenging to keep up with. As a reviewer I am wondering...is this a required change, a change just for change, or a new feature change?